src/main/scala/cromwell/Main.scala

```diff
@@ -36,19 +42,20 @@ object Main extends App {
    * Also now passing args to runAction instead of the constructor, as even sbt seemed to have issues with the args
    * array becoming null in "new Main(args)" when used with: sbt 'run run ...'
    */
-  new Main().runAction(args)
+  val AskTimeout = Timeout(5 seconds)
+  new Main().runAction(args)(AskTimeout)
 }
 
-class Main private[cromwell](enableSysExit: Boolean, managerSystem: () => WorkflowManagerSystem) {
+class Main private[cromwell](enableTermination: Boolean, managerSystem: () => WorkflowManagerSystem) {
```

Comment:
I used to do that in Java all the time since Java doesn't have keyword arguments, but given that Scala does have them...